modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -78,6 +78,13 @@ public class UpgradeCatchAllCheck {
 		_commerceAccountHelper.getCurrentCommerceAccount(null);
 	}
 
+	public void method(
+		Indexer<?> indexer, ServiceReference<Indexer<?>> serviceReference) {
+
+		Map<Indexer<?>, Indexer<?>> indexerRequests = new LinkedHashMap<>();
+		List<Indexer<?>> indexers = portlet.getIndexerInstances();
+	}
+
```

Comment:
@TamyrisTorres I already sent this on another PR to Brian, it's in his queue. So,  when merged this will be discarded. But good job noticing that!